src/main/java/com/untamedears/JukeAlert/listener/JukeAlertListener.java

```diff
@@ -539,6 +539,8 @@ private void handleSnitchEntry(Player player) {
 						lastNotifyMoveFailure = System.currentTimeMillis();
 					}
 				}
+				//Removes the snitches so they can hit the immuneToSnitch condition
+				inList.removeAll(tempList);
```

Comment:
Kind of incredible. Will have to review back to the add of cull timer code to see why the ordering is the way it is; hack seems OK otherwise (and TreeSet is fine choice).